DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: mbbs
 Title: Data and data processing functions for the Mini-Bird Breeding Survery
-Version: 0.3.0.0
+Version: 2022.0
```

Comment:
Just noting that if this PR is merged before #31 then we'll need to change the version number in #31. If #31 is merged first, then we're fine.

Reply:
Fixed in c97579dcb406611d51e59ec8f87cac815df3d2c2